src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs

```diff
@@ -136,6 +136,7 @@ public void TestInvalidDisksSpaceProperties()
         }
 
         [Fact, OuterLoop]
+        [ActiveIssue(1355, PlatformID.Windows)]
```

Comment:
You're right. I thought I tested it without it first and it still was failing but checked again now and it did not run it. (probably I looked at the previous run)